src/passes/I64ToI32Lowering.cpp

```diff
@@ -109,6 +109,12 @@ struct I64ToI32Lowering : public WalkerPass<PostWalker<I64ToI32Lowering>> {
       high->name = makeHighName(curr->name);
       module->addGlobal(high);
     }
+    auto* highBits = new Global();
```

Comment:
lgtm, but please add a comment here about what his is or how it is used (unless it's documented somewhere else that I missed).

Reply:
Done!